app/assets/v2/js/pages/kill_bounty.js

```diff
@@ -24,9 +24,31 @@ window.onload = function() {
           .removeAttr('disabled');
 
         $.each($(form).serializeArray(), function() {
-          data[this.name] = this.value;
+          if (this.value) {
+            data[this.name] = this.value;
+          }
         });
 
+        const selectedRadio = $('input[name=canceled_bounty_reason]:checked').val();
+        let reasonCancel;
+
+        if (selectedRadio == 'other') {
+          reasonCancel = $('#reason_text').val();
+        } else {
+          reasonCancel = selectedRadio;
+        }
+        var payload = {
```

Comment:
update to `const` ?

Reply:
*let ? :P

Reply:
Sorry I confused with line 33, my bad, you are right